packages/client/src/views/PrintCertificate/usePrintableCertificate.ts

```diff
@@ -156,29 +156,26 @@ export const usePrintableCertificate = (declarationId: string) => {
     const certificate = draft.data.registration.certificates[0]
     const eventDate = getEventDate(draft.data, draft.event)
     if (!isPrintInAdvance) {
-      if (isFreeOfCost(draft.event, eventDate, registeredDate, offlineData)) {
-        certificate.payments = {
-          type: 'MANUAL' as const,
-          amount: 0,
-          outcome: 'COMPLETED' as const,
-          date: new Date().toISOString()
-        }
-      } else {
-        const paymentAmount = calculatePrice(
-          draft.event,
-          eventDate,
-          registeredDate,
-          offlineData
-        )
-        certificate.payments = {
-          type: 'MANUAL' as const,
-          amount: Number(paymentAmount),
-          outcome: 'COMPLETED' as const,
-          date: new Date().toISOString()
-        }
+      const paymentAmount = calculatePrice(
+        draft.event,
+        eventDate,
+        registeredDate,
+        offlineData
+      )
+      certificate.payments = {
+        type: 'MANUAL' as const,
+        amount: Number(paymentAmount),
+        outcome: 'COMPLETED' as const,
+        date: new Date().toISOString()
       }
     }
 
+    const svg = await compileSvg(
+      offlineData.templates.certificates[draft.event].definition,
+      { ...draft.data.template, print: true },
```

Comment:
Reckon we should do an explicit `print: false` or even `print: false, preview: true` for preview? That got me also thinking if we should only have `preview: boolean` which could be more descriptive than `print`. What do you think?